pycine/file.py

```diff
@@ -25,8 +27,9 @@ def read_header(cine_file):
             "{}q".format(header["cinefileheader"].ImageCount), f.read(header["cinefileheader"].ImageCount * 8)
         )
 
-    return header
+        header = taggedBlock(cine_file, header)    #add tagged block to header dict
```

Comment:
Let's just call this function `read_tagged_blocks` and lose the comment.

Reply:
yes thats also fine for me